envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdaterTest.kt

```diff
@@ -105,6 +125,38 @@ class SnapshotUpdaterTest {
             .withoutClusters()
     }
 
+    @ParameterizedTest
+    @MethodSource("configurationModeNotSupported")
+    fun `should not generate group snapshots for modes not supported by the server`(
+        adsSupported: Boolean,
+        xdsSupported: Boolean,
+        mode: CommunicationMode
+    ) {
+        val adsGroup = AllServicesGroup(communicationMode = mode)
```

Comment:
with `XDS` mode, is it really `adsGroup`?

Reply:
my bad